### PR TITLE
[judgels-dev/judgels#4] change username field to usernameOrEmail

### PR DESCRIPTION
**Pull request by kevinluvian**
judgels-dev/judgels#4

### CODE REVIEW

**Comment by kevinluvian** [comment]
knp lintnya error ya :/

**Comment by nathanajah** [comment]
Lu uda setup prettier? :/

File: src/routes/jophiel/login/LoginPage/LoginPage.test.tsx [hunk 1]
```diff
@@ -29,8 +29,8 @@ describe('LoginPage', () => {
   });
 
   test('login form', () => {
-    const username = wrapper.find('input[name="username"]');
-    username.simulate('change', { target: { value: 'user' } });
+    const usernameOrEmail = wrapper.find('input[name="usernameOrEmail"]');
+      usernameOrEmail.simulate('change', { target: { value: 'user' } });
```
Comment (nathanajah) [inline src/routes/jophiel/login/LoginPage/LoginPage.test.tsx]: Probably this line (harusnya kalo run prettier beres sih)

File: src/routes/jophiel/login/modules/loginActions.ts [hunk 2]
```diff
@@ -18,7 +18,7 @@ export const loginActions = {
       }
 
       const user = await userAPI.getMyself(session.token);
-      toastActions.showToast(`Welcome, ${username}.`);
+      toastActions.showToast(`Welcome, ${usernameOrEmail}.`);
```
Comment (fushar) [inline src/routes/jophiel/login/modules/loginActions.ts]: `user.username`

File: src/routes/jophiel/login/LoginPage/LoginPage.test.tsx (hunk 1 above)
Comment (fushar) [reply src/routes/jophiel/login/LoginPage/LoginPage.test.tsx]: Iya ini ada leading space, makanya jadi error

**Pull request merged**